MergeCalendarsTogether.gs

```diff
@@ -37,6 +39,9 @@ const OBFUSCATE_LIST_REGEXES = [
 // should we copy event descriptions?
 const USER_INCLUDE_DESC = false;
 
+// should we copy the original attendance status from the primary calendar?
```

Comment:
```suggestion
// should we copy the original attendance status from the primary calendar?
// If set to true, we'll keep your declined/pending status instead of marking you busy.
```

Reply:
@tomsaleeba how's that?